recipes/intbitset/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "intbitset" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d4ceba6c252223a2e60072b8e42b867004eb31972098e0d16924cf0d27aed387
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - six
+  run:
+    - python
+    - six
+
+test:
+  imports:
+    - intbitset
+
+about:
+  home: http://github.com/inveniosoftware/intbitset/
+  license: GNU Lesser General Public v3 or later (LGPLv3+)
```

Please use the SPDX classifier 
```suggestion
  license: LGPL-3.0-or-later
```